reducer/reducer.go

```diff
@@ -19,7 +19,7 @@ var (
 	ErrFieldRequired = errors.New("field parameter required")
 )
 
-type Maker func() Interface
+type Maker func(*resolver.Context) Interface
```

Comment:
This isn't a git conflict but runMany() in reducer_test.go also needs updating... else the build will fail after merging this.

Reply:
Fixed